docker-x64/Dockerfile

```diff
@@ -1,12 +1,18 @@
 FROM 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker:20.10
 
 ARG PYTHON_VERSION=3.12.6
+ARG GO_VERSION
+ARG GO_SHA256_LINUX_AMD64
 ARG CI_UPLOADER_VERSION=2.38.1
 ARG CI_UPLOADER_SHA=4e56d449e6396ae4c7356f07fc5372a28999aacb012d4343a3b8a9389123aa38
 ARG VAULT_VERSION=1.17.2
 ARG VAULT_CHECKSUM=a0c0449e640c8be5dcf7b7b093d5884f6a85406dbb86bbad0ea06becad5aaab8
 ARG VAULT_FILENAME="vault_${VAULT_VERSION}_linux_amd64.zip"
 
+# Environment
+ENV GOPATH /go
+ENV GO_VERSION $GO_VERSION
```

Comment:
This is already done further in the same `Dockerfile` line 55 and 56: https://github.com/DataDog/datadog-agent-buildimages/pull/713/files#diff-5dab46fa1b83d99a3786c557458cf2336c672cbb10fbe95dbce71879d05ae715R55

We don’t need to set the same environment variables twice.

```suggestion
```

Reply:
Nice catch, thanks !